### PR TITLE
Bump Libplanet:sloth

### PR DESCRIPTION
**Pull request by OnedgeLee**
Bump Libplanet to nightly build for sloth ([prerelease-5.0.0](https://github.com/planetarium/libplanet/tree/5.0.0-alpha%2B1718170149)) and adjust APIs

### CODE REVIEW

File: Lib9c.sln [hunk 1]
```diff
@@ -68,7 +68,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Lib9c.Plugin", ".Lib9c.Plug
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Lib9c.Plugin.Shared", ".Lib9c.Plugin.Shared\Lib9c.Plugin.Shared.csproj", "{76F6C25E-94D2-4EA9-B88D-0249F44D1D16}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Libplanet.Mocks", ".Libplanet\Libplanet.Mocks\Libplanet.Mocks.csproj", "{8BC561CB-91EF-4290-893F-025E9E6A0CF7}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Libplanet.Mocks", ".Libplanet\Libplanet.Mocks\Libplanet.Mocks.csproj", "{8BC561CB-91EF-4290-893F-025E9E6A0CF7}"
```
Comment (greymistcube) [inline Lib9c.sln:71]: Why is this updated? 🤔

**Review by greymistcube — APPROVED** [review]

File: Lib9c.sln (hunk 1 above)
Comment (OnedgeLee) [reply Lib9c.sln:71]: I think this happened again...
- https://github.com/dotnet/project-system/issues/1821

I'll amend this commit

Comment (OnedgeLee) [reply Lib9c.sln]: It's quite annoying, but no idea how to fix it.
Once I suffer from like this, this will happen every time when I load this project :(

Comment (greymistcube) [reply Lib9c.sln]: It isn't a big deal I suppose. If it doesn't change multiple times, changed reference might be okay as well. 😶

**Review by greymistcube — APPROVED** [review]

File: Lib9c.sln (hunk 1 above)
Comment (OnedgeLee) [reply Lib9c.sln]: Hope this bug fixed someday...

**Review by tyrosine1153 — APPROVED** [review]

**Review by boscohyun — APPROVED** [review]

**Pull request merged**